task_02/xieyu_wang/graphic.cpp

```diff
@@ -0,0 +1,106 @@
+#include<iostream>
+#include<fstream>
+#include<cstdlib>
+const int maxDots = 14;
```

Comment:
Есть разные `naming convention` для глобальных констант, но в любом случае, они должны отличаться от обычных переменных. Давай договоримся всегда добавлять префикс `k`
```suggestion
constexpr int kMaxDots = 14;
```

Также лучше использовать `constexpr` вместо `const`, потому что `constexpr` означает что эта константа будет рассчитана во время компиляции, а не выполнения программы, что лучше.